setup.py

```diff
@@ -75,6 +75,8 @@ def run(self):
     "win-unicode-console>=0.5; sys_platform == 'win32'",
     "pywin32>=225; sys_platform == 'win32'",
     "networkx>=2.1,<2.4",
+    "speedcopy>=2.0.1",
+    "pyfastcopy>=1.0.3",
```

Comment:
Both packages are not available in conda, so this PR will break our conda package 🙁Would you be willing to submit and maintain those packages in conda? Another simpler option is to make these packages optional, so that our System.copy can work with or without those packages. So pypi would be faster than conda package.

Reply:
It is not a dealbreaker, as we could help out with those packages too, we have quite some experience with conda-forge already. Just asking. 🙂

Reply:
Also, these deps should be conditional, i.e. none are needed in Python 3.8/Linux

Reply:
Hi @efiop, I won't have enough time to maintain conda packages as well, I will let you guys make the call as to whether that is worthwhile - not forgetting that this code will potentially be deleted under Python 3.8.

I have updated the patch to fall back to standard code if packages are missing.

Reply:
@rxxg No worries. Thanks for the PR! We'll try to create and maintain those conda packages ourselves.